spec/amp-actions-and-events.md

```diff
@@ -604,12 +617,12 @@ The actions below are supported in the following AMP video elements: `amp-video`
     <td>Unmutes the video.</td>
   </tr>
   <tr>
-    <td><code>fullscreenenter</code></td>
+    <td><code>fullscreen</code></td>
```

Comment:
We want to keep `fullscreenenter` and leave `fullscreen` out: https://github.com/ampproject/amphtml/blob/master/src/service/video-manager-impl.js#L234-L240